net/rsyslog/Makefile

```diff
@@ -37,6 +37,7 @@ define Package/rsyslog/conffiles
 endef
 
 CONFIGURE_ARGS+= \
+	--enable-omprog \
```

Comment:
It actually should be separated to a another commit and probably be configurable in a submenu. I'll remove for now.